client/src/modules/shopping-list/index.jsx

```diff
@@ -6,9 +6,30 @@ import { withRouter } from 'react-router-dom';
 import ProductsContainer from 'modules/shopping-list/components/ProductsContainer';
 import { getShoppingList } from 'modules/shopping-list/model/selectors';
 import InputBar from 'modules/shopping-list/components/InputBar';
-import { fetchItemsFromGivenList } from 'modules/shopping-list/model/actions';
+import {
+  fetchItemsFromGivenList,
+  deleteList
+} from 'modules/shopping-list/model/actions';
+import DropdownMenu from 'common/components/DropdownMenu';
+import DialogBox from 'common/components/DialogBox';
+import EditIcon from 'assets/images/pen-solid.svg';
+import RemoveIcon from 'assets/images/trash-alt-solid.svg';
+import InviteUserIcon from 'assets/images/user-plus-solid.svg';
 
 class ShoppingList extends Component {
+  constructor(props) {
+    super(props);
+    this.listMenu = [
```

Comment:
please create getter for this like:
get listMenu() {
  return [...]
}

and get rid off constructor